using-disks.tex

```diff
@@ -0,0 +1,368 @@
+
+The MEGA65 also includes a ``virtual'' disk drive that can mount D81 disk image files\index{D81 disk image} stored on the SD card. Most MEGA65 software that you download from the Internet is in the form of a D81 disk image. You can create a new D81 disk image directly from the MEGA65, and start saving your BASIC programs to the SD card without any additional hardware. You can also copy files between physical floppy disks and D81 disk images.
+
+The Intro Disk Menu\index{Intro Disk} that you saw when you first switched on the computer is a program on a D81 disk image, a file named {\tt MEGA65.D81} on the SD card. The MEGA65 is initially configured to boot this disk image automatically. You can change this in the Configuration Utility.\index{Configuration!Utility} (Refer back to chatper \vref{cha:configuringyourmega}.)
+
+You can manage disk drives and virtual disk images from the Freezer menu.\index{Freezer menu} Some of these operations can be performed with BASIC commands such as {\bf MOUNT}.\index{BASIC 65 Commands!MOUNT}
+
+\subsection{Unit Numbers and Drive Numbers}
+\index{Disk Drives!Terminology}
+\index{Connections!IEC}
+
+Each disk drive (physical or virtual) is accessed via a {\it unit} number.\index{Unit number (IEC devices)} With vintage Commodore computers, the unit number refers to an IEC device connected to the computer. Commodore reserved unit numbers in the range 0 -- 31 for devices of various purposes, with 8 -- 11 reserved for disk drives. If you've ever used a Commodore 64 and typed \screentext{LOAD "*",8,1}, the ``8'' refers to the disk drive connects as unit 8. BASIC 65 disk commands use unit 8 by default, and accept a {\bf U} parameter to change it, such as: \screentext{DLOAD "MYPROGRAM",U9}\index{BASIC 65 Examples!DLOAD}\index{BASIC 65 Examples!LOAD}
+
+With the MEGA65, you can assign a unit number to the virtual disk drive with a D81 disk image mounted, or to the internal 3.5" floppy drive. You must mount a disk image or the internal 3.5" floppy drive to a unit number before it can be used. Any message sent to a unit number assigned to a virtual disk or the internal floppy drive is handled by the MEGA65. All other messages are sent to the IEC serial port.
+
+Disk commands also accept an optional parameter to specify a {\it drive} number.\index{Drive number (IEC devices)} This is only needed when connecting a vintage dual floppy drive via the IEC port, such as the Commodore 4040, 8050, or 8250. Every disk drive assigns drive number 0 to the first drive. Dual-drive units assign a drive number of 1 to the second drive. Dual disk drives are usually equipped with an IEEE-488 interface, and need an IEEE-488 to IEC converter to be used on the MEGA65. BASIC 65 disk commands use device 0 by default, and accept a {\bf D} parameter to change it.
```

Comment:
> ... use device 0 by default

Should this say **drive 0** instead of 'device 0'?

I admit I'm not entirely clear on the distinction between drive/device and we might've had past discussions in discord on this topic

Reply:
Correct, this should be "drive." "device" is ambiguous. (For a while I was confused that "drive" referred to the 0/1 that selected the internal managed drive in the Freezer menu. I had to dig around to realize that this only applies to vintage dual drives. — I just bought a vintage dual drive last weekend! I doubt it'll work with the MEGA65 due to IEC timing issues but it'll be fun to try it in GO64 mode.)